cmd/kube-proxy/app/server.go

```diff
@@ -230,7 +230,6 @@ func NewOptions() *Options {
 // Complete completes all the required options.
 func (o *Options) Complete() error {
 	if len(o.ConfigFile) == 0 && len(o.WriteConfigTo) == 0 {
-		klog.InfoS("Warning, all flags other than --config, --write-config-to, and --cleanup are deprecated, please begin using a config file ASAP")
```

Comment:
Please add the reason for the deletion and a link to the original issue.

Reply:
You mean here where the log was removed ?

Reply:
Yes, there is a reason for removing it, the config file is currently in the alpha stage and when it graduates, this warning will be added back.

Reply:
done in 34fca77c359